tests/auth_tests/test_management.py

```diff
@@ -41,15 +47,24 @@ def getpass(prompt=b'Password: ', stream=None):
                     return inputs['password']
 
             def mock_input(prompt):
+                # This must be interpolated at runtime rather than during
+                # module loading.
+                MOCK_INPUT_KEY_TO_PROMPTS['username'][1] %= {
+                    'default_username': get_default_username(),
```

Comment:
I had no idea you could do `%=`, interesting! This does feel a bit too special cased though, couldn't you add a lambda function to the `MOCK_INPUT_KEY_TO_PROMPTS`, i.e: `['Username: ', lambda: "Username (leave blank to use '%s'): " % get_default_username()]`, then call it if needed below?

It's not that important though.

Reply:
Good suggestion, updated.